package.json

```diff
@@ -14,7 +14,7 @@
     "res": "res"
   },
   "dependencies": {
-    "@httptoolkit/osx-find-executable": "^1.1.0",
+    "@httptoolkit/osx-find-executable": "^2.0.1",
```

Comment:
Have you tested this on a Mac? This is a breaking change. You'll need to change every usage of the osx-find-executable module in the code here to match this, or it will completely break all browser detection on Mac. See https://github.com/httptoolkit/osx-find-executable/commit/3c8e1d9193efa3c20cab18f80e1d44408a6d8ad0